modules/base/include/modules/base/processors/datasource.h

```diff
@@ -126,57 +128,63 @@ class DataSource : public Processor {
 };
 
 template <typename DataType, typename PortType>
-DataSource<DataType, PortType>::DataSource(DataReaderFactory* rf, std::string_view file,
+DataSource<DataType, PortType>::DataSource(DataReaderFactory* rf, std::string_view filePath,
                                            std::string_view content)
     : Processor()
+    , filePath{"filename", "File", filePath, content}
```

Comment:
To make the Python api consistent with the c++ api (could break code in many places and requires a network update): 
```suggestion
    , filePath{"filePath", "File", filePath, content}
```
Not sure if the naming refactoring of the property is worth the trouble. Should be done in a separate PR

Reply:
that was by accident. Will be reverted.